### ISSUE TITLE
About the validation set

### ISSUE THREAD

**Issue by xdg988** [zhawe01/fairseq-gec#22]
Thanks for your excellent work! 
After running train.sh, I found that there were about 5,000 valid samples. Since there are only 1381 sentences in CoNLL-2013 test set, can you please tell me where those valid samples come from?

**Comment by zhawe01** [comment]
We separated 5000 examples from the train dataset as our valid dataset.
